src/drivers/types/cached-cookie.ts

```diff
@@ -0,0 +1,6 @@
+import Cached from './cached';
+import { CookieValueTypes } from 'cookies-next/lib/types';
+
+export default interface CachedCookie extends Cached {
+  value: CookieValueTypes;
+}
```

Comment:
As this is specific to the `next-json-cookie` driver, I think this should live in the same folder.

Reply:
yeh ok